src/d3a/d3a_core/redis_connections/aggregator_connection.py

```diff
@@ -62,15 +62,16 @@ def aggregator_callback(self, payload):
 
     def _select_aggregator(self, message):
         if message['aggregator_uuid'] not in self.aggregator_device_mapping:
-            msg = f"{message['aggregator_uuid']} aggregator not found."
-            error_response_message = {
-                "status": "error", "aggregator_uuid": message['aggregator_uuid'],
+            self.aggregator_device_mapping[message['aggregator_uuid']] = []
```

Comment:
this will overwrite the list every time. So only one device can be connected to an aggregator.

Reply:
This case would only exist when there was no aggregator in first place, so no way it would overwrite since the aggregator_device never existed with the targeted aggregator_uuid.

Reply:
True. Sorry. Understood.